jsonrpc/package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "@sourcegraph/vscode-jsonrpc",
 	"description": "A json rpc implementation over streams",
-	"version": "3.3.0-alpha.1",
+	"version": "3.3.0-alpha.1.sourcegraph.2",
```

Comment:
Okay, makes sense. Still looks a bit weird by semver conventions, but I guess it's the only way if we don't want to uncouple the versioning. Btw latest upstream version is alpha.2